### DISCUSSION TITLE
Old bug with "too long" data works again

### DISCUSSION THREAD

**Discussion by sinKettu** [DefectDojo/django-DefectDojo#4991] [Q&A] [answered]
Hello!
I chose "Feature request" issue type, but this is more likely a question.
There was a PR [#1804](https://github.com/DefectDojo/django-DefectDojo/pull/1804) with fix for parser for ZAP reports. It fixed handling a length of some fields from the reports. This commit appeared it version 1.8 of Defect Dojo, but disappeared in versions 2.x.x. Due to this situation, the [old bug](https://github.com/DefectDojo/django-DefectDojo/issues/1720) works now.
Do you have any plans to return fixes from the PR I mentioned?

**Comment by StefanFl** [comment, marked as answer]
This has been addressed by #4974

**Comment by sinKettu** [comment]
Thank you!
